typescript/cli/src/tests/commands/helpers.ts

```diff
@@ -44,6 +44,39 @@ export const WARP_CORE_CONFIG_PATH_2 = `${REGISTRY_PATH}/deployments/warp_routes
 
 export const DEFAULT_E2E_TEST_TIMEOUT = 100_000; // Long timeout since these tests can take a while
 
+export enum KeyBoardKeys {
+  ARROW_DOWN = '\x1b[B',
+  ARROW_UP = '\x1b[A',
+  ENTER = '\n',
+  TAB = '\t',
+}
+
+export async function asyncStreamInputWrite(
+  stream: NodeJS.WritableStream,
+  data: string | Buffer,
+): Promise<void> {
+  stream.write(data);
+  // Adding a slight delay to allow the buffer to update the output
+  await sleep(500);
+}
+
+export async function selectAnvil2AndAnvil3(
+  stream: ProcessPromise,
+): Promise<void> {
+  // Scroll down through the mainnet chains list and select anvil2
+  await asyncStreamInputWrite(
+    stream.stdin,
+    `${KeyBoardKeys.ARROW_DOWN.repeat(3)}${KeyBoardKeys.TAB}`,
```

Comment:
will this change as we add more chains?

Reply:
it shouldn't because we override the registry for the tests and we only have `anvil1`, `anvil2` and `anvil3` in the custom registry for the tests